tests/unit-tests/webhooks/sequelize/webhooksFormatter-test.js

```diff
@@ -0,0 +1,233 @@
+const sinon = require('sinon');
```

Comment:
can remove file, this file is in master under
`unit-tests/webhooks/webhooksformatter-test.js`